README.md

```diff
@@ -32,6 +32,8 @@ Here are the current Endpoints that are supported:
 
 - Instance Profiles Service
 
+- SCIM (preview mode, subject to changes)
```

Comment:
can we raise the middle pom semantic version for this change?

Reply:
I thought that versioning is done automatically by the release plugin - should I change something in the  pom.xml  file?

Reply:
so versioning update is done automatically for the patch version (smallest number). But for semantic versioning changes that has to be done manually. In this case, this is a full feature addition, so to help users who will in the future depend on SCIM api, we want to make sure its easy to track which version it was released on.

So all you need to do is change the pom version from 2.4.2-SNAPSHOT to 2.5.1-SNAPSHOT